sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -404,7 +404,7 @@ object FunctionRegistry {
     expression[Month]("month"),
     expression[MonthsBetween]("months_between"),
     expression[NextDay]("next_day"),
-    expression[CurrentTimestamp]("now", true),
+    expression[Now]("now"),
```

Comment:
Since the usage is different between `now` and `current_timestamp`, I added a `Now` expr.

Reply:
btw, we don't need to add a version where an alias added? For example, `current_timestamp` and `now` have been implemented in the different versions, 1.5 and 1.6.

Reply:
What is the difference between `now` and `current_timestamp`? Should `now` extend `Nondeterministic`, for example?

Reply:
`CURRENT_TIMESTAMP` is a SQL syntax and `NOW` is not: https://github.com/apache/spark/blob/master/sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4#L777